<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerForm</class>
 <widget class="QWidget" name="CustomerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="customer_tab_widget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>29</y>
     <width>1001</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="catalog_tab">
    <attribute name="title">
     <string>Catalog</string>
    </attribute>
    <widget class="QTableWidget" name="catalog_table_widget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>451</width>
       <height>521</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Retail price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Discount price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="new_order_button">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>New order</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="new_order_table_widget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>421</width>
       <height>491</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cost</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="submit_new_order_button">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_new_order_button">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_cost_label">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>0</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Cost:</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_cost_label_value">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="orders_tab">
    <attribute name="title">
     <string>My orders</string>
    </attribute>
    <widget class="QComboBox" name="status_filter_comboBox">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="status_filter_label">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter by status:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="orders_table_widget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>571</width>
       <height>521</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Order number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cost</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="details_table_widget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>130</y>
       <width>301</width>
       <height>411</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Product Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cost</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="order_number_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Details of Order: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="hide_details_button">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
